_posts/2023-05-10-solidity-0.8.20-release-announcement.md

```diff
@@ -0,0 +1,80 @@
+ * SMTChecker: Group all messages about unsupported language features in a single warning. The CLI option ``--model-checker-show-unsupported`` and the JSON option ``settings.modelChecker.showUnsupported`` can be enabled to show the full list.
+ * SMTChecker: Properties that are proved safe are now reported explicitly at the end of analysis. By default, only the number of safe properties is shown. The CLI option ``--model-checker-show-proved-safe`` and the JSON option ``settings.modelChecker.showProvedSafe`` can be enabled to show the full list of safe properties.
+ * Standard JSON Interface: Add experimental support for importing ASTs via Standard JSON.
+ * Yul EVM Code Transform: If available, use ``push0`` instead of ``codesize`` to produce an arbitrary value on stack in order to create equal stack heights between branches.
+
+
+### Bugfixes
+ * ABI: Include events in the ABI that are emitted by a contract but defined outside of it.
+ * Immutables: Disallow initialization of immutables in try/catch statements.
+ * SMTChecker: Fix false positives in ternary operators that contain verification targets in its branches, directly or indirectly.
+
+
+### AST Changes
+ * AST: Add the ``internalFunctionIDs`` field to the AST nodes of contracts containing IDs of functions that may be called via the internal dispatch. The field is a map from function AST IDs to internal dispatch function IDs. These IDs are always generated, but they are only used in via-IR code generation.
+ * AST: Add the ``usedEvents`` field to ``ContractDefinition`` which contains the AST IDs of all events emitted by the contract as well as all events defined and inherited by the contract.
+
+As always, a big thank you to all contributors who helped make this release possible!
+
+## How to install/upgrade
+To upgrade to the latest version of the Solidity Compiler, 
+simply follow the [installation instructions](https://docs.soliditylang.org/en/latest/installing-solidity.html) available in our documentation. 
+Our team has made sure to provide detailed and straightforward steps to make the upgrade process as seamless as possible. 
+If you have any questions or run into any issues during the upgrade process, 
+don't hesitate to reach out to our [community Matrix channel](https://matrix.to/#/#ethereum_solidity:gitter.im). 
+
+You can download the new version of Solidity here: [v0.8.20](https://github.com/ethereum/solidity/releases/tag/v0.8.20).
+If you want to build from the source code, do not use the source archives generated automatically by GitHub, 
+instead please use [`solidity_0.8.20.tar.gz`](https://github.com/ethereum/solidity/releases/download/v0.8.20/solidity_0.8.20.tar.gz)
+and take a look at [our documentation on how to build from source](https://docs.soliditylang.org/en/v0.8.20/installing-solidity.html#building-from-source)
```

Comment:
Not sure what to do about this now, since the link will be broken :-D. Maybe best to use
```suggestion
and take a look at [our documentation on how to build from source](https://docs.soliditylang.org/en/latest/installing-solidity.html#building-from-source)
```